keyboard-navigation.js.tid

```diff
@@ -15,11 +15,37 @@ exports.name = "tw5-keyboard-navigation"; exports.after = ["rootwidget"];
 
 exports.startup = function () {
 
+// Keybinds section
+const navigate_up_key = 'k'
+const navigate_down_key = 'j'
+const bindings = {
+	more: null,
+	info: null,
+	'new-here': null,
+	'new-journal-here': null,
+	clone: null,
+	'export-tiddler': null,
+	edit: 'e',
+	delete: 'd',
+	permalink: null,
+	permaview: null,
+	'open-window': null,
+	'close-others': null,
+	close: 'c',
+	'fold-others': null,
+	fold: null
+}
+// if you just want to add/change keybindings, you don't need to look further
+
+
+const BTN_CLASSNAME_PREFIX = 'tc-btn-%24%3A%2Fcore%2Fui%2FButtons%2F' // tiddler buttons have classnames in the form {BTN_CLASSNAME_PREFIX}{button name}
+const INSTANT_NAVIGATION = false; // all transitions with {navigate_up_key} and {navigate_down_key} are instant
+const INSTANT_NAVIGATION_BEFORE = 800; // if navigated before x milliseconds, transition is instant
+
 const MARK_CURRENT_TIDDLER = true;
 // tiddler's top position should be +/- LIMIT pixels in order to be considered the topmost
 const LIMIT = 50;
 
-var ignoreKeyboardLayout = false;
```

Comment:
Why do you remove this option? This is a regression.

Reply:
Yes sorry about this! I at first didn't see it used (as some of the e.code lines were commented out). I will bring this back, shouldn't have removed it